src/coffee/connect/rest.coffee

```diff
@@ -119,7 +119,15 @@ class Rest
   # Throws an {Error} if `access_token` could not be retrieved
   _preRequest: (params, callback) ->
     _req = (retry) =>
-      unless @_options.access_token
+      isTokenExpired = false
+      if @_tokenExpiration
+        expirationTime = Date.now() + (@_tokenExpiration * 1000) -
```

Comment:
No, what you get from the API when you request an `access_token` is the `expires_in` value (2 days).
The `_tokenExpiration` is that value.

What I'm doing here is to calculate the **future** date, which consists in:

```
now + expirationPeriod - gap
```